Gemfile.lock

```diff
@@ -450,7 +450,8 @@ GEM
       rack (~> 2.2)
       rack-protection (= 2.2.2)
       tilt (~> 2.0)
-    sqlite3 (1.4.4)
+    sqlite3 (1.5.0)
+      mini_portile2 (~> 2.8.0)
```

Comment:
Hmm. I haven't into this closely yet, but it might impact things if it requires extra build steps.

Reply:
I don't _think_ this will impact Pro/Omnibus

https://github.com/sparklemotion/sqlite3-ruby/commit/4bf6f66222696bf2037da1e667c8f385b9322205

Reply:
`mini_portile2` is already required by `nokogiri`.